documentation/book/proc-configuring-kafka-connect-authentication-scram-sha-512.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-connect-authentication.adoc
+
+[id='proc-configuring-kafka-connect-authentication-scram-sha-512-{context}']
+= Configuring SCRAM-SHA-512 authentication in Kafka Connect
+
+.Prerequisites
+
+* An {ProductPlatformName} cluster
+* A running Cluster Operator
+
+.Procedure
+
+. Find out the username of the user which should be used for authentication.
```

Comment:
Find out the username of the user which should be used for authentication. - I think we can move this text to the Prerequisites section and rewrite as "Username of the user which should be used for authentication"?